.all-contributorsrc

```diff
@@ -266,6 +266,15 @@
       "contributions": [
         "code"
       ]
+    },
+    {
+      "login": "b00tk1ll",
+      "name": "Guilherme Silveira",
+      "avatar_url": "https://avatars2.githubusercontent.com/u/8213622?v=4",
+      "profile": "http://twitter.com/b00tk1ll",
+      "contributions": [
+        "code"
+      ]
```

Comment:
Sorry, but remove this part, because I'll add you as a contributor.

I always do this when I release.

Reply:
Ok, I made the change